packages/amplify-opensearch-simulator/index.js

```diff
@@ -0,0 +1,293 @@
+const { fromEvent } = require('promise-toolbox');
+const waitPort = require('wait-port');
+const detectPort = require('detect-port');
+const log = require('logdown')('opensearch-emulator');
```

Comment:
where are these logs going?

Reply:
These are printed out to the console. However, I removed the `logdown` dependency all together in favor of `amplify-prompts`.